apps/hip/models.py

```diff
@@ -1 +1,76 @@
-# Create your models here.
+from wagtail.admin.edit_handlers import FieldPanel, StreamFieldPanel
+from wagtail.core import blocks
+from wagtail.core.fields import StreamField
+from wagtail.core.models import Page
+
+
+class TableRow(blocks.StructBlock):
+    column_1 = blocks.RichTextBlock(
+        max_length=255, required=False, help_text=("Text for column 1"),
```

Comment:
Looks like [black wants us to re-format this](https://github.com/caktus/philly-hip/runs/1855971661?check_suite_focus=true). Should be able to fix this by running `pre-commit run -a` locally.